articles/153_roslaunch_remote_launch.md

```diff
@@ -0,0 +1,94 @@
+# Multi-Machine and Remote Launching
+This article describes the design of the components of the ros2 launch system related to the remote execution of processes.
+
+## Goals
+ROS 1's launch system provided the means to launch nodes remotely via SSH by defining a "machine" and referencing that machine in the declaration of each node meant to run on it.
+The machine declaration would contain the information necessary to execute processes remotely, such as address, username, password, etc.
+The goal of this design is to enable the same ability to launch nodes on remote machines, as well as to provide infrastructure for more advanced capabilities.
+
+- Monitor the status and manage lifecycles of nodes across systems
+- Shutdown nodes remotely
+- Ability to detach the machine used to launch a system and have the system continue running
+    - Be able to reconnect to the system to monitor and manage nodes running remotely
+
+
+To achieve our goals, this update will consist of two primary components: an ExecuteRemote action, and a LaunchServiceNode.
```

Comment:
Based on the re-scope of this design - it seems like this "Goals" section shouldn't be talking about the `LaunchServiceNode` since it's not specified within this design. If I remember correctly, we established that the `LaunchServiceNode` was independently interesting and designable. I do see that we want to talk about how these two things will interact, but I'm not sure if that belongs here in this doc anymore.